packages/models/src/read-models/tenantReadModel.ts

```diff
@@ -57,7 +59,27 @@ export const TenantMailReadModel = TenantMail.extend({
   createdAt: z.string().datetime(),
 });
 
+export const TenantFeatureCertifierReadModel = TenantFeatureCertifier;
+export type TenantFeatureCertifierReadModel = z.infer<
+  typeof TenantFeatureCertifierReadModel
+>;
+
+export const TenantFeatureDelegatedProducerReadModel =
+  TenantFeatureDelegatedProducer.extend({
+    availabilityTimestamp: z.string().datetime(),
+  });
+export type TenantFeatureDelegatedProducerReadModel = z.infer<
+  typeof TenantFeatureDelegatedProducerReadModel
+>;
+
+export const TenantFeatureReadModel = z.discriminatedUnion("type", [
+  TenantFeatureCertifierReadModel,
+  TenantFeatureDelegatedProducerReadModel,
+]);
+export type TenantFeatureReadModel = z.infer<typeof TenantFeatureReadModel>;
+
 export const TenantReadModel = Tenant.extend({
+  features: z.array(TenantFeatureReadModel),
```

Comment:
Okay, only after this do I have doubts about the previous comments. Do you intend to allow more different features for the tenant? In this case features sounds too generic and not related to a person like Tenant; in this case I suggest to use "grant" or "permission" to identify tenant allowed to this features.